File: XamarinRepairApp/XamarinRepairApp/Model/RoomModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Json;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;

using XamarinRepairApp.Utils;

namespace XamarinRepairApp.Model
{
    public class RoomModel
    {
        private JsonValue mData;

        public RoomModel(JsonValue jsonValue)
        {
            mData = jsonValue;
        }

        public int GetId()
        {
            return Helper.GetInt(mData["ID"]);
        }

        public String GetTitle()
        {
            return Helper.GetString(mData["Title"]);
        }
    }
}

File: AndroidRepairApp/app/src/main/java/com/canviz/repairapp/data/RoomModel.java
package com.canviz.repairapp.data;

import org.json.JSONException;
import org.json.JSONObject;

public class RoomModel {
    private final JSONObject mData;

    public RoomModel(JSONObject data) {
        mData = data;
    }

    public int getId(){
        try{
            return mData.getInt("ID");
        }catch(JSONException e) {
            return 0;
        }
    }

    public String getTitle(){
        try{
            return mData.getString("Title");
        }catch(JSONException e) {
            return null;
        }
    }
}
